atlascope/core/management/populate/datasets.json

```diff
@@ -1,28 +1,34 @@
 [
     {
-        "name": "Local Vandy Dataset",
-        "id": 201,
-        "description": "This is the test dataset from the VANDY Explorer",
+        "name": "HTA9_1_BA_F_ROI02",
```

Comment:
Can we apply a name to these that describes the content of the image somehow? And that makes the difference between the two tif datasets easier to discern?

Reply:
I don't think it's a good idea, just because of the confusion in "provenance" it will cause when we go back to the source to explore other data.